CHANGELOG.md

```diff
@@ -4,6 +4,11 @@
 
 RC. IN-PROGRESS
 
+* **feat** remove `enableNavigationTracking` and have the behavior as the default
+* **feat** `getRating` has been consolidated into `getVitalsScore`
+* **chore** rename `incrementUjNavigation` to `trackUJNavigation`
```

Comment:
Note for next PR, it is more professional to also link the PR that makes the changes, and explain the why of this change. Same for the others.